packages/@dcl/sdk-commands/src/commands/deploy/index.ts

```diff
@@ -123,23 +122,25 @@ export async function main(options: Options) {
   const authChain = Authenticator.createSimpleAuthChain(entityId, address, signature)
 
   // Uploading data
-  const catalyst = await getCatalyst(options.args['--target'], options.args['--target-content'])
+  const { client, url } = await getCatalyst(options.args['--target'], options.args['--target-content'])
 
-  printProgressInfo(options.components.logger, `Uploading data to: ${catalyst.getContentUrl()}...`)
+  printProgressInfo(options.components.logger, `Uploading data to: ${url}...`)
 
   const deployData = { entityId, files: entityFiles, authChain }
   const position = sceneJson.scene.base
-  const network = chainId === ChainId.ETHEREUM_GOERLI ? 'goerli' : 'mainnet'
+  const network =
+    chainId === ChainId.ETHEREUM_GOERLI ? 'goerli' : chainId === ChainId.ETHEREUM_SEPOLIA ? 'sepolia' : 'mainnet'
```

Comment:
are we going to still support goerli ? If it has an end we should remove all of it so we dont have death code here. (Just like we did with ropsten)
if we want to support the two testnets for a while, we can add an issue and remove it later.

Reply:
it's like that due to gamejam, removed now